<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>440</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_BigShifts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Bigshifts</string>
   </property>
   <widget class="QLabel" name="label_BigShifts_Advice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set slider to 1 to disable that feature (default: 11)</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_BigShifts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>372</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>41</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_BigShifts">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Overlaps">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Overlaps</string>
   </property>
   <widget class="QLabel" name="label_overlap_MDX">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_overlap_MDX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
    <property name="inputConversionRatio" stdset="0">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_overlap_MDXv3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>94</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_overlap_MDXv3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>overlap_MDX: (default: 0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>74</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>overlap_MDXv3: (default: 8)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>124</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>overlap_demucs: (default: 0.6)</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_overlap_demucs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>144</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
    <property name="inputConversionRatio" stdset="0">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_overlap_demucs">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Weights">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Weights</string>
   </property>
   <widget class="QLabel" name="label_weight_MDXv3">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_weight_MDXv3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_weight_VOCFT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>94</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_weight_VOCFT">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>weight_MDXv3: (default: 6)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>74</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>weight_VOCFT: (default: 5)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>124</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>weight_HQ3: (default: 2)</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_weight_HQ3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>144</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_weight_HQ3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Output format</string>
   </property>
   <widget class="QComboBox" name="comboBox_OutputFormat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PCM_16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLOAT</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Proceed only if you know what you are doing! &lt;/span&gt; Default values are good enough.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ExportSettings">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string comment="Export settings to a file"/>
   </property>
   <property name="text">
    <string>Export settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ImportSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string comment="Import settings from a file"/>
   </property>
   <property name="text">
    <string>Import settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_RestoreDefaults">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string comment="Export settings to a file"/>
   </property>
   <property name="text">
    <string>Restore defaults</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
